src/status_im/contexts/settings/saved_addresses/view.cljs

```diff
@@ -0,0 +1,41 @@
+(ns status-im.contexts.settings.saved-addresses.view
```

Comment:
I think it would be good to align with @mohsen-ghafouri, @smohamedjavid @seanstrom @ibrkhalil on how they plan to structure the namespaces in settings. 
It seems to me like it should be `status-im.contexts.settings.wallet.saved-addresses.view` otherwise you will run into a huge list of feaures if don't separate "settings" by context (i.e wallet in this case)

Reply:
also saved-addresses has a few parts to it alone, 
setting the name and color page, the page to enter this address etc

Reply:
Good suggestion. I'll sync with wallet settings crew re: structure.

Reply:
for profile settings we have it like this `status-im.contexts.profile.settings.view`, it will be good wallet settings follow the same structure `status-im.contexts.wallet.settings.view`

Reply:
Hmm, should settings not be its own context?

Cc @ilmotta

Reply:
<img width="384" alt="Screenshot 2024-04-24 at 11 10 51 AM" src="https://github.com/status-im/status-mobile/assets/1925158/2ff118f6-d1d7-4ffe-abb7-904d3f315ae4">

This is what I have. I feel that we need to have a top level settings context because how elaborate settings are. 
But I'd defer this decision to the Javid as he is the boss. I'm happy with either.

Reply:
I don't have any strong opinion about them, if we consider settings as a top level context then we can move profile settings here later. happy with both structure, just wanted to have consistency about settings (profile and wallet)

Reply:
Okay. In that case, I'd request to move profile to top level settings namespace. Because wallet settings run deep and it's better to isolate them.

Reply:
IMO, It should be in `status-im.contexts.settings.wallet.*` ns.

Placing it inside the wallet (`status-im.contexts.wallet.settings.*`) ns is not a good idea as settings related to the wallet have a broader scope compared to profile settings and it will crowd the wallet context with deeply nested files.

Reply:
Late in the party, but fully agree with all of you to have a separate top-level context.